Ginger/GingerCoreCommon/RunLib/Agent.cs

```diff
@@ -583,6 +589,115 @@ public override bool SerializationError(SerializationErrorType errorType, string
                     }
                     return true;
                 }
+                else if (string.Equals(name, nameof(DriverType)))
+                {
+                    if (string.Equals(value, "SeleniumChrome"))
```

Comment:
use switch here and determine the browser type in it. rest of the code is repeated, so keep it one function or use after the switch

Reply:
handled